index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Personal Portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <!-- The navigator row at the top -->
+    <header>
+      <a href="#"><h4>Raymond OLISA</h4></a>
```

Comment:
- [ ] Nice usage of `a` tag for your logo but please do not use` hx` tag inside an  `a` tag.  It wouldn't be valid code, and on some level that's bad for your search optimization. kindly do for the rest as well.